lib/tests/formulae.rb

```diff
@@ -607,51 +611,47 @@ def formula!(formula_name, args:)
         test "brew", "install", "--only-dependencies", *install_args,
               env:  { "HOMEBREW_DEVELOPER" => nil },
               args: args
+
         test "brew", "install", *install_args,
               env:  { "HOMEBREW_DEVELOPER" => nil },
               args: args
-        install_passed = steps.last.passed?
```

Comment:
Would this create too much noise if an install failed but it tried to test it anyway?

Reply:
Given @SMillerDev's comments: "too much noise on an install failure" sounds like a good thing 😁

Reply:
`/me hides`

Personally I'd rather have a clear error with some context then a cleaner log.

Reply:
Cool, put back early return on install failure.